core/pom.xml

```diff
@@ -34,6 +34,10 @@
       <artifactId>galley-api</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.commonjava.maven.galley</groupId>
```

Comment:
I'm going to go ahead and merge this PR but I'd like to follow up on this before we're completely done.